.chloggen/propagate-errors.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: otlpreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Propagate errors from the exporter back to the caller
+
+# One or more tracking issues or pull requests related to the change
+issues: [7486]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: The Collector can now propagate errors from backends to the client when used as a simple gateway.
```

Comment:
I wonder if the terminology simple gateway may confuse end users who have deployed the collector as a gateway using the recommended batch processor, which wouldn't benefit from this functionality

Reply:
Agree, perhaps "proxy" would be a better word here.